datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/http.py

```diff
@@ -0,0 +1,62 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+import click
+from datadog_checks.dev.tooling.utils import get_valid_integrations, get_check_files, get_config_spec
+
+from ..console import CONTEXT_SETTINGS, echo_info, echo_warning
+
+REQUEST_LIBRARY_FUNCTIONS = {
+    'requests.get',
+    'requests.post',
+    'requests.head',
+    'requests.put',
+    'requests.patch',
+    'requests.delete'
+}
+
+
+def validate_config_http(file, check):
+    config_valid = True
+    if os.path.isfile(file):
+        with open(file, "r") as f:
+            if 'instances/http' not in f.read():
+                echo_warning(f'Detected {check}\'s spec.yaml file does not contain `instances/http` '
+                             f'but {check} uses http wrapper')
+
+    return config_valid
+
+
+def validate_use_http_wrapper(file, check):
+    uses_http_wrapper = False
+
+    with open(file, "r") as f:
+        for num, line in enumerate(f):
+            if 'self.http' in line:
+                uses_http_wrapper = True
```

Comment:
should we return once detected instead of continuing through the entire file?

Reply:
Wasn't sure if I should handle the case where the http wrapper and request library used at the same time. For example, if there's a `self.http.get` and `requests.get` one line below it.

Reply:
Ah that's true, then that works